Xamarin.Forms.Core.UnitTests/BindableLayoutTests.cs

```diff
@@ -13,12 +15,12 @@ public class BindableLayoutTests : BaseTestFixture
 		[Test]
 		public void TracksEmpty()
 		{
-			var layout = new StackLayout
+			StackLayout layout = new StackLayout
 			{
 				IsPlatformEnabled = true,
 			};
 
-			var itemsSource = new ObservableCollection<int>();
+			ObservableCollection<int> itemsSource = new ObservableCollection<int>();
```

Comment:
Same here

Reply:
Sorry, those changes were unintentional from VS doing a cleanup. I'll fix.

Reply:
Oh, I see. No worries, Don't worry about them, it's not too much big of a deal. Thank for your help.